Makefile

```diff
@@ -105,16 +58,16 @@ uninstall:
 	@echo "$@"
 	@rm -f $(addprefix $(CMD_DESTDIR)/bin/,$(notdir $(CMD_BINARIES)))
 
-add-ltag:
-	$(shell go env GOPATH)/bin/ltag $(LTAG_TEMPLATE_FLAG) -v
-
 clean:
 	rm -rf $(OUTDIR)
 
 vendor:
 	@GO111MODULE=$(GO111MODULE_VALUE) go mod tidy
 	@cd ./cmd ; GO111MODULE=$(GO111MODULE_VALUE) go mod tidy
 
+add-ltag:
```

Comment:
Why shouldn't `add-ltag` also be moved to a script?

Reply:
will move it in next update